library/Icinga/Application/Config.php

```diff
@@ -412,17 +412,16 @@ public static function app($configname = 'config', $fromDisk = false)
      */
     public static function module($modulename, $configname = 'config', $fromDisk = false)
     {
-        if (! isset(self::$modules[$modulename])) {
-            self::$modules[$modulename] = array();
+        if (! isset(static::$modules[$modulename])) {
+            static::$modules[$modulename] = array();
```

Comment:
I'm only interested in fixing the caching issue.

Not going to nit pick, feel free to change my commit.